src/Sylius/Bundle/ResourceBundle/Controller/ResourceUpdater.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ResourceBundle\Controller;
```

Comment:
I am aware of number of classes inside controller folder of ResourceBundle, but maybe it make sense to at least put this class in separate folder?